app/Conference.php

```diff
@@ -19,6 +19,8 @@ public function meetNewFriend($username)
 
         $this->newFriends()->save($friend);
 
+        $friend->fetchAvatar();
```

Comment:
The only reason it was here was so that the avatar was available when returning to the frontend after the friend was added. If it was done as an event, we wouldn't have the avatar available straight away.